cpp/src/text/bpe/load_merge_pairs.cu

```diff
@@ -70,6 +72,7 @@ std::unique_ptr<detail::mp_table_map_type> initialize_mp_table_map(
 {
   auto mp_table_map = std::make_unique<mp_table_map_type>(
     static_cast<size_t>(input.size()),
+    cudf::detail::CUCO_DESIRED_LOAD_FACTOR,
```

Comment:
this should improve the performance a lot.

Reply:
Could we see some benchmarks on this PR?

Reply:
So what is the default value of load factor if we don't provide this?

Reply:
If not specified, the map will use the first argument, `static_cast<size_t>(input.size())` as the capacity, resulting in a fully loaded hash table.